R/get_map_points.R

```diff
@@ -8,47 +8,49 @@
 #' @param point_type \code{character}. Filter points to those of one or more
 #'   point type(s). Options are "cleavage", "bedding", "axial plane", "fault
 #'   plane", "foliation", and "joint". Ignored if `point_id` is supplied.
-#' @param min_lat \code{integer}. A minimum latitude that represents the
-#'   southwest corner of a bounding box of the points to return. Requires
-#'   `min_lng`, `max_lat`, and `max_lng`. Ignored if `point_id` is supplied.
-#' @param min_lng \code{integer}. A minimum longitude that represents the
-#'   southwest corner of a bounding box of the points to return. Requires
-#'   `min_lat`, `max_lat`, and `max_lng`. Ignored if `point_id` is supplied.
-#' @param max_lat \code{integer}. A maximum latitude that represents the
-#'   northeast corner of a bounding box of the points to return. Requires
-#'   `min_lat`, `min_lng`, and `max_lng`. Ignored if `point_id` is supplied.
-#' @param max_lng \code{integer}. A maximum longitude that represents the
-#'   northeast corner of a bounding box of the points to return. Requires
-#'   `min_lat`, `min_lng`, and `max_lat`. Ignored if `point_id` is supplied.
+#' @param min_lat \code{integer}. The minimum latitude of the points to return.
```

Comment:
I think this update to the documentation could lead to further confusion. From my perspective, it is very important to convey that these input values are being used to produce a bounding box, and all information within that bounding box is returned. I don't think this is really conveyed here. Perhaps "The minimum latitude used to construct the bounding box..."?

Reply:
I agree with describing this as a 'bounding box', but don't agree that, for example, the "maximum longitude represents the northeast corner", surely it is the eastern edge? Would describing these as the edges of the box rather than the corners be the best option?